Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -262,7 +263,9 @@ public class PythonCompiler : SinglePassCompiler {
         var ctorName = IdProtect(ctor.CompileName);
 
         // Class-level fields don't work in all python version due to metaclasses.
-        var argList = ctor.Destructors.Select(d => $"(\'{IdProtect(d.CompileName)}\', {TypeName(d.Type, wr, d.tok)})").Comma();
+        // Adding a more restrictive type would be desirable, but Python expects their definition to precede this.
+        var argList = ctor.Destructors.Where(d => !d.IsGhost)
```

Comment:
Could we maybe give a name to `Destructors.Where(d => !d.IsGhost)`?

Reply:
In `DatatypeCtor`?